docs/1.0. EDID-Mapping.md

```diff
@@ -0,0 +1,220 @@
+# AMWA BCP-XXX-YY: EDID Mapping in NMOS Systems \[Work In Progress\]
+
+
+## Background
+
+AMWA NMOS specifications provide resources for specifying receiver capabilities of an NMOS device.  Recent extension of these resources allow NMOS Controllers developed for PRO A/V User-Stories to be created.  In particular, these NMOS Controllers include Controllers developed for the AIMS sponsered IPMX set of open standards [IPMX](#ipmx-ref). 
```

Comment:
> receiver capabilities of an NMOS device

It makes me thinking that Receiver Caps belong to Device resource rather than to Receiver. Also, I think, we need use capitalization to show that we refer to the specific entity, not to some abstract capabilities.